lib/ts/xmlhttprequest.ts

```diff
@@ -70,6 +66,21 @@ export function addInterceptorsToXMLHttpRequest() {
 
         const eventHandlers: Map<keyof XMLHttpRequestEventMap, Set<XHREventListener<any>>> = new Map();
 
```

Comment:
we have changed the implementation of saveTokensFromHeaders in axios.ts and fetch.ts. Why not in this file?

Reply:
Sorry, that was a miss. I have fixed it now.